modules/ROOT/pages/salesforce/salesforce-connector-100-xml-maven-edit.adoc

```diff
@@ -0,0 +1,70 @@
+= Salesforce XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Studio XML editor or a text editor, perform the following actions to get access to the connector:
+
+Add the namespace for the connector to your XML flow.
+Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/salesforce
+http://www.mulesoft.org/schema/mule/salesforce/current/mule-salesforce.xsd
+----
+
+This example shows how the namespace statements are placed in the mule XML block:
```

Comment:
"mule XML block"? Should that be Mule, or do we mean a placeholder of `_mule_`?
```

Reply:
I will change it to <mule> XML block.